### PR TITLE
Bump Citus PGXN to 9.3.0

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: META.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.2.4",
+   "version": "9.3.0",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
-         "file": "src/backend/distributed/sql/citus--9.1-1--9.2-1.sql",
+         "file": "src/backend/distributed/sql/citus--9.1-1--9.3-0.sql",
```
Comment (onurctirtir) [inline META.json]:
```suggestion
         "file": "src/backend/distributed/sql/citus--9.3-2--9.4-1.sql",
```

File: META.json [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.2.4",
+   "version": "9.3.0",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
-         "file": "src/backend/distributed/sql/citus--9.1-1--9.2-1.sql",
+         "file": "src/backend/distributed/sql/citus--9.3-2--9.4-1.sql",
```
Comment (hanefi) [inline META.json]: At some point we should stop caring about this field I guess.

Should we just place `citus.so` her and stop modifying it maybe?

**Review by hanefi — APPROVED** [review]
1 minor comment, you can :shipit:

File: META.json (hunk 2 above)
Comment (onurctirtir) [reply META.json]: or readme.md can even make more sense

Comment (onurctirtir) [reply META.json:11]: but we don't have readme there, citus.so makes sense

Comment (onurctirtir) [inline META.json]:
```suggestion
         "file": "README.md",
```

**Pull request merged**